### PR TITLE
Made myself a helper for all those +/- number fields

### PR DESCRIPTION
**Pull request by werebus**

### CODE REVIEW

File: app/helpers/scoring_helper.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  end
+
+  def increment_button(target_field, value, type, options)
+    types = { :+ => ['plus', :max], :- => ['minus', :min] }
+    disabled = (options[types[type][1]] && value == options[types[type][1]])
+
+    button_tag types,
+               class: 'btn btn-primary increment',
+               type: :button,
+               data: { field: sanitize_to_id(target_field),
```
Comment (Anbranin) [inline app/helpers/scoring_helper.rb]: API dock is down and info on what `sanitize_to_id` is is scarce--what does it do?

Comment (werebus) [reply app/helpers/scoring_helper.rb]: That gets you e.g. from `onboard_judging[minutes_elapsed]` to `onboard_judging_minutes_elapsed`

That is, from the `name` of the field to the `id` of the field.

Comment (Anbranin) [reply app/helpers/scoring_helper.rb]: oh cool

File: lib/app_form_builder.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AppFormBuilder < ActionView::Helpers::FormBuilder
```
Comment (Anbranin) [inline lib/app_form_builder.rb]: The fact that you could do this and then specify a `builder` for the form is new to me. Sweet!

Comment (werebus) [reply lib/app_form_builder.rb]: Yeah. You can also, in your app config do something like

```ruby
ActionView::Base.default_form_builder = AppFormBuilder
```

In the past, I've used gems for Bootstrap or Foundation that do this behind the scenes.

**Pull request merged**